$NetBSD: patch-abcde,v 1.2 2015/07/01 09:28:25 wiz Exp $

Default to cdda2wav since icedax is not in pkgsrc.

--- abcde.orig	2015-06-18 12:26:40.000000000 +0000
+++ abcde
@@ -3441,7 +3441,7 @@ NEROAACTAG=neroAacTag
 
 CDPARANOIA=cdparanoia
 CD_PARANOIA=cd-paranoia
-CDDA2WAV=icedax
+CDDA2WAV=cdda2wav
 DAGRAB=dagrab
 CDDAFS=cp
 PIRD=pird
